doc/tools/docgram/orderedGrammar

```diff
@@ -648,7 +648,7 @@ ref_or_pattern_occ: [
 
 occs_nums: [
 | LIST1 [ natural | ident ]
-| "-" [ natural | ident ] LIST0 int_or_var
+| "-" LIST1 [ natural | ident ]
```

Comment:
Not that the `nat_or_var` expresses an idea different than `[ natural | ident ]`. The `nat_or_var` indicates that it is a natural or an Ltac variable denoting a natural. With `[ natural | ident ]`, isn't there a risk that one thinks at a natural or an hypothesis?

I know that the ambiguity already exists in other situations (e.g. we don't write `hyp_or_var` to mean an hypothesis or an Ltac variable denoting an hypothesis, since it is an `ident` in both cases), so maybe this is hopeless anyway.

Reply:
I'm open to eventually creating more nonterminals that just to show the semantics better, such as `var` (in both the doc and the mlgs), but that's not my priority now.  The text in the doc should make clear that the ident is for a variable.

Reply:
Interesting, this corresponds to my suggestion in https://github.com/coq/coq/pull/13343#discussion_r523794967.
@jfehrle: Your answer to this suggestion said that the `nat_or_var` nonterminal exists but was little used so spliced in the documentation. IMHO it would then be worth to:
1. use it in this case (no semantics change)
2. not splice it in the documentation (for symmetry with the `int_or_var` nonterminal).

Reply:
No, it corresponds to https://github.com/coq/coq/pull/13343#discussion_r524341931, which I mentioned in the PR description.  Splicing or not splicing `nat_or_var` is an independent issue better handled where it was raised in #13343.